recipe/meta.yaml

```diff
@@ -37,11 +37,12 @@ requirements:
     - python
     - {{ pin_compatible('numpy') }}
     - packaging
-    - psutil
+    - psutil <6.0
```

Comment:
@ajoubertza  I pinned psutil to <6.0 to fix the tests.
Failure is only due to a deprecation warning that I see you already fixed in [33c0f4](https://gitlab.com/tango-controls/pytango/-/commit/33c0f472768f3901cf2d60d19a30c81b138b418f)

Should we apply that patch instead?

Reply:
The patch would be better.  Thanks.

Reply:
Done